src/pages/ALLO_ManageAllocations.page

```diff
@@ -226,17 +224,16 @@
                                     </div>
                                 </div>
                             </td>
-                            <td>
-                            </td>
-                            <td>
-                                <apex:commandButton styleClass="slds-button slds-button_neutral slds-button_small" value="{!$Label.alloAddRow}" id="addRowBTN" title="{!$Label.alloAddRow}" action="{!addRow}" reRender="theTable" immediate="true" rendered="{!listAlloSize==0}">
-                                    <span class="slds-form-element__label slds-assistive-text"><apex:outputText value="{!$Label.alloAddRowAtPosition}"/> {!cnt}</span>
-                                    <apex:param name="rowForNew" value="{!cnt}" assignTo="{!rowNumber}"/>
-                                </apex:commandButton>
-                            </td>
+                            <td colspan="2"></td>
                         </tr>
                    </tbody>
                 </table>
+
+                 <!-- Add Row -->
+                <div class="slds-p-around_medium">
+                        <apex:commandLink action="{!addRow}" immediate="true" reRender="theTable, messages" 
+                            value="{!$Label.alloAddRow}" title="{!$Label.alloAddRow}" />
+                </div>
```

Comment:
@sam-knox the new add action just says "Add Row" as a link text.  Not sure if you think it needs to be more descriptive?

Reply:
@patrick-yan-sf - Add Row works just fine!